cloud/src/common/config.h

```diff
@@ -229,6 +229,8 @@ CONF_mInt64(max_s3_client_retry, "10");
 // Max byte getting delete bitmap can return, default is 1GB
 CONF_mInt64(max_get_delete_bitmap_byte, "1073741824");
 
+CONF_mInt64(max_txn_commit_byte, "7340032");
```

Comment:
add comment, why it's not 8.3MB (described by the following code comment)?

Reply:
@zhannngchen done